### PR TITLE
Fix credit for bensheldon

### PR DESCRIPTION
**Pull request by etiennebarrie**
https://contributors.rubyonrails.org/contributors/ben-sheldon/commits
https://contributors.rubyonrails.org/contributors/ben-sheldon-he/commits

cc @bensheldon

### CODE REVIEW

File: app/models/names_manager/canonical_names.rb [hunk 1]
```diff
@@ -238,6 +238,7 @@ def self.map(canonical_name, *also_as)
     map 'Ben Holley',                 'benolee'
     map 'Ben Murphy',                 'benmmurphy'
     map 'Ben Sandofsky',              'sandofsky'
+    map 'Ben Sheldon',                'Ben Sheldon [he'
```
Comment (bensheldon) [inline app/models/names_manager/canonical_names.rb:241]:
```suggestion
    map 'Ben Sheldon',                'Ben Sheldon'
```

**Review by bensheldon — COMMENTED** [review]
Thank you!

**Comment by fxn** [comment]
Hey! Let me investigate this a little bit. Mappings are meant for "real" aliases, so to speak, this is more of a parsing/false positive issue, things between square brackets have special handling due to credit conventions in commit messages.

I'll write back!

**Comment by fxn** [comment]
Credits fixed by https://github.com/rails/rails-contributors/commit/7e21a358f211bb6eef33d7761ff5ee66bf83a3e5.

@bensheldon your credits have been consolidated, they are all in your page https://contributors.rubyonrails.org/contributors/ben-sheldon/commits ❤️.

@etiennebarrie thanks for noticing, and for the patch anyway!!!

**Pull request closed without merging**